internal/jobqueue/jobqueue.go

```diff
@@ -56,7 +56,7 @@ type JobQueue interface {
 	// finished, respectively.
 	//
 	// If the job is finished, its result will be returned in `result`.
-	JobStatus(id uuid.UUID, result interface{}) (queued, started, finished time.Time, canceled bool, err error)
```

Comment:
A note about this function returning also the deps would be nice.

Reply:
Fixed.